src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write your code here
+const thead = document.querySelector('thead').querySelector('tr');
+const tfoot = document.querySelector('tfoot').querySelector('tr');
+const tbodytr = document.querySelector('tbody').querySelectorAll('tr');
+
+const newHeader = document.createElement('th');
+
+newHeader.textContent = 'Position';
+
+const newFooter = document.createElement('th');
+
+newFooter.textContent = 'Position';
+
+thead.insertBefore(newHeader, thead.lastElementChild);
+tfoot.insertBefore(newFooter, tfoot.lastElementChild);
+
+tbodytr.forEach((item) => {
+  const bodytTh = document.createElement('td');
+
+  bodytTh.textContent = item.cells[1].innerText;
+  item.insertBefore(bodytTh, item.lastElementChild);
+});
```

Comment:
try to simplify this logic using the `.cloneNode`

feel free to ask for some help in the chat